src/N98/Magento/Command/Admin/User/LockCommand.php

```diff
@@ -0,0 +1,88 @@
+<?php
+
+namespace N98\Magento\Command\Admin\User;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Input\InputArgument;
+
+class LockCommand extends AbstractAdminUserCommand
+{
+    /**
+     * The number of days to lock for (by default)
+     * @var int
+     */
+    const LIFETIME_DEFAULT = 31;
+
+    /**
+     * Setup
+     *
+     * @return void
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('admin:user:lock')
+            ->addArgument('username', InputArgument::REQUIRED, 'Admin username to lock')
+            ->addArgument('lifetime', InputArgument::OPTIONAL, 'Optional - lock lifetime (default one month)')
```

Comment:
I think it's worth to tell the unit of measurement in the parameter description, like:

`'Optional - lock lifetime in seconds (default one month)'`